prsa/statistical_analysis.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
```

Comment:
In my opinion - not neeede- the aim is to calculate PRSA - data analysis is not necessery (in our project I calculate it in different manner)